src/lTerm_widget.mli

```diff
@@ -253,6 +256,58 @@ class ['a] radiobutton : 'a radiogroup -> string -> 'a -> object
 
 end
 
+(** {6 Scrollbars} *)
+
+(** Adjustable integer value from (0..range-1) *)
+class type adjustment = object
+
+  method range : int
+    (** range of adjustment *)
+
+  method set_range : int -> unit
+    (** set range of adjustment.
+    
+     The range can be changed dynamically, set statically in an
+     [initializer] if known when the widget is created,
+     or set in the [set_allocation] method if the size of the 
+     widget needs to be known. *)
+
+  method offset : int
+    (** offset from (0..range-1) *)
+
+  method set_offset : int -> unit
+    (** Set offset clipped to range.
+    
+    The scrollbars will call [queue_draw] when [set_offset]
+    is called. *)
+
+  method incr : unit
+    (** increment offset by one step 
+    
+    If range > number of scroll bar steps then step>=1. *)
+
+  method decr : unit
+    (** decrement offset by one step *)
+
+  method scroll_bar_size : int
```

Comment:
I'm not sure this should be here, the idea is that an adjustment is not necessarily a scroll_bar